_data/tutorials.yaml

```diff
@@ -670,3 +670,22 @@ sliding-windows:
     ruby: disabled
     scala: disabled
     swift: disabled
+
+error-handling:
+  title: "Handling uncaught exceptions"
+  meta-description: "Handling uncaught exceptions"
+  slug: "/error-handling"
+  question: "How do I handle uncaught exceptions in my streams application?"
```

Comment:
@bbejeck is there a chance a ksqlDB app would be written for this?  If so, would it make sense to generalize the question with the `streams` word?